src/Table/Table.component.tsx

```diff
@@ -59,13 +59,14 @@ interface TableProps
         FontWeightProps,
         BorderRadiusProps,
         BorderProps,
-        WidthProps,
         MaxWidthProps,
         MinWidthProps,
         SpaceProps {
     borderRadius?: string;
     background?: string;
     altBackground?: string;
+    width?: any;
+    height?: any;
```

Comment:
Are these both really `any`? It seems like `string` based on the stories.

Reply:
It's a responsive prop, so it could be a `string` or an `object` (or even a `number`). There was a TypeError here (on master, unrelated to PR) that was pretty painful, so I chose not to fight it.